### ISSUE TITLE
Deprecated features used

### ISSUE THREAD

**Issue by 0x022b** [dev-sec/chef-os-hardening#230]
**Describe the bug**
Running the latest os-hardening cookbook with the lastest Chef client I get the following warning:

```
Deprecated features used!
Resource sysctl_param from a cookbook is overriding the resource from the client. Please upgrade your cookbook or remove the cookbook from your run_list before the next major release of Chef.
```

**Expected behavior**
The run should finish without errors.

**OS / Environment**

This has been verified to happen on CentOS 7 and Ubuntu 18.04.

**Chef Version**

```paste below
Chef: 14.6.47
```

**Cookbook Version**

```paste below
os-hardening: 3.1.0
```

**Additional context**
The sysctl cookbook has been deprecated and now shipping as part of Chef 14. It will continue to function for Chef 13 users, but will not be updated.

**Comment by artem-sidorenko** [comment]
@0x022b thanks for raising this issue! This is related to #228 and will be fixed in the near future with a next major release.

**Comment by marcomoscatiello** [comment]
got in the same issue too using chef 14.8 , hope it get sorted soon

**Comment by artem-sidorenko** [comment]
Resolved with #228. I want to have/add some other cleanups and refactorings to the new major release, it will take a little time to see this released.
